NEWS

```diff
@@ -282,7 +282,7 @@ CHANGES WITH 249 in spe:
         * A new ConditionOSRelease= setting has been added to unit files to
           check os-release(5) fields. The "=", "!=", "<", "<=", ">=", ">"
           operators may be used to check if some field has some specific value
-          or do a aphanumerical comparison. Equality comparisons are useful for
+          or do the alphanumerical comparison. Equality comparisons are useful for
```

Comment:
"a" is correct here, "the" is out of place.

Reply:
so, "an"? instead of "a"?

Reply:
"an". Sorry!